Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian
+FROM alpine:3.5
```

Comment:
Alpine Linux uses musl in deprecation of glibc. I've been using Alpine Linux for all my workloads, including Go and Java ones with no issues for the past year.